web_ui/frontend/app/cache/page.tsx

```diff
@@ -67,6 +67,7 @@ export default function Home() {
                                             type: 'time',
                                             time: {
                                                 round: 'second',
+                                                minUnit: 'minute'
```

Comment:
```suggestion
                                                minUnit: 'second'
```
We might still want to have second-level precision (not millisecond though) so that admin can get a higher granularity

Reply:
Since I cap the resolution and time range at 1 minute I don't think it makes sense to make the precision show anything higher.

Reply:
If you think I should add more precision to this field then that sounds good to me though, lets slot that for the next release.